css/Draft.css

```diff
@@ -133,7 +136,7 @@
 .public-DraftStyleDefault-orderedListItem.public-DraftStyleDefault-listLTR:before {
   left: -36px;
   position: absolute;
-  text-align: right;
+  text-align: center;
```

Comment:
@anuradha9712  Text alignment of the serial numbers should be right to keep the actual text of list items aligned to same vertical line.